package.json

```diff
@@ -59,7 +59,7 @@
     "@babel/preset-react": "^7.0.0",
     "@babel/preset-typescript": "^7.3.3",
     "@openmrs/esm-api": "^3.0.8",
```

Comment:
Great - I'd just update all the (core) dependencies to their latest version (3.1.1).